application/controllers/timers.py

```diff
@@ -0,0 +1,90 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.timer import Timer
+from application import db
+
+timer_fields = {
+    'id': fields.Integer,
+    'work_interval': fields.String,
+    'rest_interval': fields.String
+}
+
+timer_list_fields = {
+    'count': fields.Integer,
+    'timers': fields.List(fields.Nested(timer_fields)),
+}
+
+timer_post_parser = reqparse.RequestParser()
+timer_post_parser.add_argument(
+    'work_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='work_interval parameter is required'
+)
+timer_post_parser.add_argument(
+    'rest_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='rest_interval parameter is required'
+)
+
+class TimersResource(Resource):
+    def get(self, timer_id=None):
+        if timer_id:
+            timer = Timer.query.filter_by(id=timer_id).first()
+            return marshal(timer, timer_fields)
```

Comment:
Not really sure about what marshal is about. I might make an issue card for that as well.